packages/components/accordion/test/accordion.spec.ts

```diff
@@ -1,36 +1,34 @@
-import { mount } from '@vue/test-utils'
-import { describe, it, expect } from 'vitest'
-import PuikAccordionGroup from '../src/accordion-group.vue'
-import PuikAccordion from '../src/accordion.vue'
-import type { MountingOptions, VueWrapper } from '@vue/test-utils'
-
-let wrapper: VueWrapper<any>
-const factory = (template: string, options: MountingOptions<any> = {}) => {
+import { mount, ComponentMountingOptions, VueWrapper } from '@vue/test-utils';
+import { describe, it, expect } from 'vitest';
+import { PuikAccordion, PuikAccordionGroup } from '@prestashopcorp/puik-components';
+
+let wrapper: VueWrapper<any>;
+const factory = (template: string, options: ComponentMountingOptions<any> = {}) => {
   wrapper = mount({
     components: {
       'puik-accordion-group': PuikAccordionGroup,
-      'puik-accordion': PuikAccordion,
+      'puik-accordion': PuikAccordion
     },
     template,
-    ...options,
-  })
-}
-
-export const getAccordion = (wrapper) => wrapper.findComponent(PuikAccordion)
-export const getAccordionContainer = (component) =>
-  component.find('.puik-accordion')
-export const getAccordionContent = (component) =>
-  component.find('.puik-accordion__content')
-export const getAccordionHeader = (component) =>
-  component.find('.puik-accordion__header')
-export const getAccordionTitle = (component) =>
-  component.find('.puik-accordion__header__content__title')
-export const getAccordionSubTitle = (component) =>
-  component.find('.puik-accordion__header__content__sub-title')
-export const getAccordionExpandIcon = (component) =>
-  component.find('.puik-accordion__header__expand__icon')
-export const getAccordionIcon = (component) =>
-  component.find('.puik-accordion__header__icon')
+    ...options
+  });
+};
+
+export const getAccordion = (wrapper: VueWrapper<any>) => wrapper.findComponent(PuikAccordion);
+export const getAccordionContainer = (component: VueWrapper<any>) =>
+  component.find('.puik-accordion');
+export const getAccordionContent = (component: VueWrapper<any>) =>
+  component.find('.puik-accordion__content');
+export const getAccordionHeader = (component: VueWrapper<any>) =>
+  component.find('.puik-accordion__header');
+export const getAccordionTitle = (component: VueWrapper<any>) =>
+  component.find('.puik-accordion__header__content__title');
+export const getAccordionSubTitle = (component: VueWrapper<any>) =>
+  component.find('.puik-accordion__header__content__sub-title');
+export const getAccordionExpandIcon = (component: VueWrapper<any>) =>
+  component.find('.puik-accordion__header__expand__icon');
+export const getAccordionIcon = (component: VueWrapper<any>) =>
```

Comment:
Yes but i didn't refactored this part, in this PR i only focused on the Monorepo and some global fixes, in this case i think we should the same factorisation as the other tests using the wrapper variable instead of passing it as a function param

Reply:
Ok, we'll do it in another PR